src/lib/components/menu/Menu.svelte

```diff
@@ -3,11 +3,15 @@
   const globalActiveMenu = writable(undefined);
 </script>
 
-<script>
-  import { createEventDispatcher, setContext, onMount } from "svelte";
+<script lang="ts">
+  import { guidGenerator } from "$lib/util/guid";
+  import { createEventDispatcher, onMount, setContext } from "svelte";
   import { writable } from "svelte/store";
   import { fade } from "svelte/transition";
-  import { guidGenerator } from "$lib/util/guid";
+
+  export let dark = false;
+  setContext("rill:menu:dark", dark);
```

Comment:
I would be in favor of this being `"rill:theme:dark"`. If the prop is undefined (should be default) it doesn't set this flag. If it is set, it will `setContext`

this will enable the user to specify the theme in a parent

the default can be `false` if there is no parent context setting it

this will require a change to `MenuItem` too to consume this + have a similar default

Reply:
Ok that makes sense. The reason I had it this way was because we currently have some dark menus, some light menus. But for a consistent dark/light theme, we should only go with one or the other all throughout the app?